frontend/src/scenes/insights/InsightTabs/InsightDisplayConfig.tsx

```diff
@@ -116,6 +123,13 @@ export function InsightDisplayConfig({ filters, activeView, disableTable }: Insi
                     </span>
                 )}
 
+                {activeView === InsightType.TRENDS &&
+                !filters.breakdown_type &&
+                !filters.compare &&
+                (!filters.display || filters.display === ACTIONS_LINE_GRAPH_LINEAR) ? (
```

Comment:
Could we also add this to the cumulative chart... or not really worth it?

Reply:
We decided go narrow on this PR, to reduce chances of further iterations.

Reply:
And also, not as relevant